### PR TITLE
[Feature] need to collect FAQ issues to official website faq.md docs

### PR DESCRIPTION
**Pull request by AhahaGe**
[Feature] need to collect FAQ issues to official website faq.md docs, [issue 3020](https://github.com/apache/incubator-dolphinscheduler/issues/3020)

### CODE REVIEW

**Comment by yangyichao-mango** [comment]
Good job.

I think we also can refer to the Chinese Translation specifications [1] of Apache Flink.
For example, there should be space between Chinese and English to make the document clearer.

[1] https://cwiki.apache.org/confluence/display/FLINK/Flink+Translation+Specifications

**Comment by samz406** [comment]
Before version 1.2, it was escheduler. After version 1.2, it was DolphinScheduler. This needs to be explained.

1.2版本之前是escheduler 1.2版本后是dolphinScheduler，这个需要说明下

File: docs/zh-cn/release/faq.md [hunk 1]
```diff
@@ -275,3 +275,266 @@ A ： 将 hive pom
...
+
+---
+
+## Q：调度任务不能上线
+A：1，我们可以成功创建调度任务，并且表t_scheduler_schedules中也成功加入了一条记录，但当我点击上线后，前端页面无反应且会把t_escheduler_schedules这张表锁定，我测试过将t_escheduler_schedules中的RELEASE_state字段手动更新为1这样前端会显示为上线状态。
+
+---
+
+## Q：请问swagger ui的地址是什么
+A：1，http://${api server ip}:${ api server port}/escheduler/doc.html?language=zh_CN&lang=cn。
```
[the remark this answers is not in the document]
Comment (AhahaGe) [reply docs/zh-cn/release/faq.md:488]: Good Idea.

File: docs/zh-cn/release/faq.md [hunk 2]
```diff
@@ -275,3 +275,266 @@ A ： 将 hive pom
...
+## Q：如何改变Master服务的启动端口
+<p align="center">
+   <img src="https://user-images.githubusercontent.com/8263441/62352160-0f3e9100-b53a-11e9-95ba-3ae3dde49c72.png" width="60%" />
+ </p>
+A：1，修改application_master.properties配置文件，例如：server.port=12345。
+
+---
+
+## Q：调度任务不能上线
+A：1，我们可以成功创建调度任务，并且表t_scheduler_schedules中也成功加入了一条记录，但当我点击上线后，前端页面无反应且会把t_escheduler_schedules这张表锁定，我测试过将t_escheduler_schedules中的RELEASE_state字段手动更新为1这样前端会显示为上线状态。
```
[the remark this answers is not in the document]
Comment (AhahaGe) [reply docs/zh-cn/release/faq.md:483]: OK

**Comment by AhahaGe** [comment]
> Before version 1.2, it was escheduler. After version 1.2, it was DolphinScheduler. This needs to be explained.
> 
> 1.2版本之前是escheduler 1.2版本后是dolphinScheduler，这个需要说明下

ok

**Comment by AhahaGe** [comment]
> Good job.
> 
> I think we also can refer to the Chinese Translation specifications [1] of Apache Flink.
> For example, there should be space between Chinese and English to make the document clearer.
> 
> [1] https://cwiki.apache.org/confluence/display/FLINK/Flink+Translation+Specifications

Yes, I will follow this specification.

**Comment by AhahaGe** [comment]
> Good job.
> 
> I think we also can refer to the Chinese Translation specifications [1] of Apache Flink.
> For example, there should be space between Chinese and English to make the document clearer.
> 
> [1] https://cwiki.apache.org/confluence/display/FLINK/Flink+Translation+Specifications

fixed, please check.

**Comment by AhahaGe** [comment]
> > Before version 1.2, it was escheduler. After version 1.2, it was DolphinScheduler. This needs to be explained.
> > 1.2版本之前是escheduler 1.2版本后是dolphinScheduler，这个需要说明下
> 
> ok

fixed, please check.

File: docs/zh-cn/release/faq.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (AhahaGe) [reply docs/zh-cn/release/faq.md]: fixed, please check.

File: docs/zh-cn/release/faq.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (AhahaGe) [reply docs/zh-cn/release/faq.md]: fixed , please check.

**Comment by yangyichao-mango** [comment]
Hi,
There are some users ask for api docs, I think we can add this to FAQ. Thx a lot~
Refer to https://github.com/apache/incubator-dolphinscheduler/issues/3231

**Comment by AhahaGe** [comment]
> Hi,
> There are some users ask for api docs, I think we can add this to FAQ. Thx a lot~
> Refer to [apache/incubator-dolphinscheduler#3231](https://github.com/apache/incubator-dolphinscheduler/issues/3231)

added, but I suggest we can continuously provide faq, but not all in this pr.

File: docs/zh-cn/release/faq.md [hunk 3]
```diff
@@ -275,3 +281,271 @@ A ： 将 hive pom
...
+   <img src="https://user-images.githubusercontent.com/51871547/80302626-b1478d00-87dd-11ea-97d4-08aa2244a6d0.jpg" width="60%" />
+ </p>
+<p align="center">
+   <img src="https://user-images.githubusercontent.com/51871547/80302626-b1478d00-87dd-11ea-97d4-08aa2244a6d0.jpg" width="60%" />
+ </p>
+A：这个 [bug](https://github.com/apache/incubator-dolphinscheduler/issues/1477)  描述了问题的详情。这个问题在 1.2.1 版本已经被修复了。
+对于 1.2.1 以下的版本，这种情况的一些提示：
+
+```
+1。清空 zk 下这个路径的任务：/dolphinscheduler/task_queue
```
Comment (yangyichao-mango) [inline docs/zh-cn/release/faq.md:397]:
```suggestion
1. 清空 zk 下这个路径的任务：/dolphinscheduler/task_queue
```

File: docs/zh-cn/release/faq.md [hunk 4]
```diff
@@ -275,3 +281,271 @@ A ： 将 hive pom
...
+
+---
+
+## Q：调度系统设置了一个秒级的任务，导致系统挂掉
+A：调度系统不支持秒级任务。
+
+---
+
+## Q：编译前后端代码 (dolphinscheduler-ui) 报错不能下载"https://github.com/sass/node-sass/releases/download/v4.13.1/darwin-x64-72_binding.node"
+A：1，cd dolphinscheduler-ui 然后删除 node_modules 目录
```
Comment (yangyichao-mango) [inline docs/zh-cn/release/faq.md:429]:
```suggestion
A：cd dolphinscheduler-ui 然后删除 node_modules 目录 
```

File: docs/zh-cn/release/faq.md [hunk 5]
```diff
@@ -275,3 +281,271 @@ A ： 将 hive pom
...
+
+---
+   
+## Q：生产环境部署方式有推荐的最佳实践吗
+A：1，如果没有很多任务要运行，出于稳定性考虑我们建议使用 3 个节点，并且最好把 Master/Worder 服务部署在不同的节点。如果你只有一个节点，当然只能把所有的服务部署在同一个节点！通常来说，需要多少节点取决于你的业务，海豚调度系统本身不需要很多的资源。充分测试之后，你们将找到使用较少节点的合适的部署方式。
+
+---
+
+## Q：DEPENDENT 节点
+A：1，DEPENDT 节点实际是没有执行体的，是专门用来配置数据周期依赖逻辑，然后再把执行节点挂载后面，来实现任务间的周期依赖。
```
Comment (yangyichao-mango) [inline docs/zh-cn/release/faq.md:476]:
```suggestion
A：1，DEPENDENT 节点实际是没有执行体的，是专门用来配置数据周期依赖逻辑，然后再把执行节点挂载后面，来实现任务间的周期依赖。
```

File: docs/zh-cn/release/faq.md [hunk 6]
```diff
@@ -275,3 +281,271 @@ A ： 将 hive pom
...
+---
+
+## Q：后续任务在前置任务失败情况下仍旧可以执行
+A：在启动工作流的时候，你可以设置失败策略：继续还是失败。
+![设置任务失败策略](https://user-images.githubusercontent.com/15833811/80368215-ee378080-88be-11ea-9074-01a33d012b23.png)
+
+---
+
+## Q：工作流模板 DAG、工作流实例、工作任务及实例之间是什么关系 工作流模板 DAG、工作流实例、工作任务及实例之间是什么关系
+A：1，一个 dag 支持最大并发 100？是指产生 100 个工作流实例并发运行吗？
```
Comment (yangyichao-mango) [inline docs/zh-cn/release/faq.md:319]:
```suggestion
Q：1，一个 dag 支持最大并发 100？是指产生 100 个工作流实例并发运行吗？
```

File: docs/zh-cn/release/faq.md (hunk 5 above)
Comment (AhahaGe) [reply docs/zh-cn/release/faq.md:476]: ok

File: docs/zh-cn/release/faq.md (hunk 3 above)
Comment (AhahaGe) [reply docs/zh-cn/release/faq.md:397]: no, but change other 2. 3. to 2。 3。

File: docs/zh-cn/release/faq.md (hunk 5 above)
Comment (AhahaGe) [reply docs/zh-cn/release/faq.md]: fixed

File: docs/zh-cn/release/faq.md (hunk 6 above)
Comment (AhahaGe) [reply docs/zh-cn/release/faq.md]: fiexed

**Pull request merged**